python/ray/tests/test_streaming_generator_regression.py

```diff
@@ -0,0 +1,127 @@
+import time
```

Comment:
You need to add it to bazel BUILD file otherwise it won't run by CI.

Do you want to add it to `test_streaming_generator_4.py` or create a `_5.py`?

Reply:
Good call!